### PR TITLE
Star Cult Timer 

### PR DESCRIPTION
**Pull request by efefury**
integrated in the Dwarven Mines Overlay

:pray: not another 10 cleanup commits

### CODE REVIEW

**Review by jani270 — DISMISSED** [review]
Please add yourself to the 2.1.1.md

**Comment by kr45732** [comment]
Very nice code efe!

File: src/main/java/io/github/moulberry/notenoughupdates/options/seperateSections/Mining.java [hunk 1]
```diff
@@ -138,11 +138,12 @@ public class Mining {
...
 			"\u00a73Gemstone Powder: \u00a7d6,243",
 			"\u00a73Forge 1) \u00a79Diamonite\u00a77: \u00a7aReady!",
-			"\u00a73Pickaxe CD: \u00a7a78s"
+			"\u00a73Pickaxe CD: \u00a7a78s",
+			"\u00a73Star Cult In: \u00a7a78s"
 		}
 	)
 	@ConfigAccordionId(id = 2)
-	public List<Integer> dwarvenText2 = new ArrayList<>(Arrays.asList(0, 1, 2, 3, 4));
+	public List<Integer> dwarvenText2 = new ArrayList<>(Arrays.asList(0, 1, 2, 3, 4,5));
```
Comment (kr45732) [inline src/main/java/io/github/moulberry/notenoughupdates/options/seperateSections/Mining.java:146]: you should add a space between the comma and 5

**Review by hannibal002 — APPROVED** [review]

**Comment by romangraef** [comment]
While we are here: Integration with the CalendarOverlay ( https://github.com/NotEnoughUpdates/NotEnoughUpdates/blob/master/src/main/java/io/github/moulberry/notenoughupdates/miscgui/CalendarOverlay.java#L93 )

**Review by romangraef — APPROVED** [review]
i guess that goes in my backlog then

**Review by Lulonaut — APPROVED** [review]

**Pull request merged**